[2021-02-18 22:43:05,158] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:42:00+00:00 [queued]>
[2021-02-18 22:43:05,164] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:42:00+00:00 [queued]>
[2021-02-18 22:43:05,164] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:43:05,164] {taskinstance.py:1018} INFO - Starting attempt 1 of 2
[2021-02-18 22:43:05,164] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:43:05,174] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): get_tweets_by_user_list> on 2021-02-18T22:42:00+00:00
[2021-02-18 22:43:05,176] {standard_task_runner.py:51} INFO - Started process 18685 to run task
[2021-02-18 22:43:05,181] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'tweets_by_user', 'get_tweets_by_user_list', '2021-02-18T22:42:00+00:00', '--job-id', '1141', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/tweets_by_user.py', '--cfg-path', '/tmp/tmph0sy1phu']
[2021-02-18 22:43:05,184] {standard_task_runner.py:76} INFO - Job 1141: Subtask get_tweets_by_user_list
[2021-02-18 22:43:05,212] {logging_mixin.py:103} INFO - Running <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:42:00+00:00 [running]> on host ip-172-31-3-59.ec2.internal
[2021-02-18 22:43:05,238] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=tweets_by_user
AIRFLOW_CTX_TASK_ID=get_tweets_by_user_list
AIRFLOW_CTX_EXECUTION_DATE=2021-02-18T22:42:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-18T22:42:00+00:00
[2021-02-18 22:43:05,239] {logging_mixin.py:103} INFO - Autenticação no twitter
[2021-02-18 22:43:05,255] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:43:05,407] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:43:05,919] {logging_mixin.py:103} INFO - 200
[2021-02-18 22:43:05,928] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:43:06,096] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:43:06,581] {logging_mixin.py:103} INFO - 400
[2021-02-18 22:43:06,589] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:43:06,748] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:43:07,088] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:43:07,092] {logging_mixin.py:103} INFO - 0:00:01.853508
[2021-02-18 22:43:07,092] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:43:08,997] {logging_mixin.py:103} INFO - 0:00:01.904821
[2021-02-18 22:43:09,025] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:43:09,047] {logging_mixin.py:103} INFO - 0:00:00.049844
[2021-02-18 22:43:09,047] {logging_mixin.py:103} INFO - 600 devidamente inseridas no banco
[2021-02-18 22:43:09,048] {python.py:118} INFO - Done. Returned value was: None
[2021-02-18 22:43:09,052] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=tweets_by_user, task_id=get_tweets_by_user_list, execution_date=20210218T224200, start_date=20210218T224305, end_date=20210218T224309
[2021-02-18 22:43:09,074] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-18 22:43:09,086] {local_task_job.py:118} INFO - Task exited with return code 0
